### PR TITLE
fix: Add missing input warning to symbolic and big o input elements

### PR DESCRIPTION
**Pull request by eliotwrobson**
Removed unused branches from mustache templates, per #10429. All of the other elements I checked actually had code that used this template branch, so I'm not sure if we want to remove that code from those. These two elements handle the missing submission error differently, which is why that branch was unused.

### CODE REVIEW

**Comment by jonatanschroeder** [comment]
Both of these elements actually update a `missing_input` variable, only they don't use this variable anywhere. Would it make more sense to actually pass the value of this variable to the template so that a reasonable error is shown in this case?

**Comment by jonatanschroeder** [comment]
To test this:
* Open a question that has the element;
* In developer tools, change the name of the element's input to something different;
* Save&Grade.

**Comment by eliotwrobson** [comment]
Without this key being set, this is the current error message output by this element:

image

I think this is an adequate error message, given the one being removed here is saying almost the same thing, and it's pretty difficult to distinguish between an answer not being submitted and the question code changing like this.

**Comment by prairielearn1engr** [comment]
I do think it would benefit us to aim for consistency, if only so that we don't have to pick between these two options the next time we introduce an element. I'm pretty indifferent as to which option we actually pick.

**Comment by nwalters512** [comment]
I do think it would benefit us to aim for consistency, if only so that we don't have to pick between these two options the next time we introduce an element. I'm pretty indifferent as to which option we actually pick.

**Comment by eliotwrobson** [comment]
Changed this to now be consistent with other elements. We might want to open up a follow-up issue to make the missing input button look like the new invalid input button, but that should be done later across all elements for consistency (also, this case is not common).

Screenshots:

Symbolic Input:
image

Big O Input:
image

File: apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache [hunk 1]
```diff
@@ -83,13 +83,15 @@
             data-content="{{parse_error}}"
         > Invalid <i class="fa fa-exclamation-triangle" aria-hidden="true"></i></a>
     {{/parse_error}}
-
     {{#missing_input}}
+        ${{type}}($
```
Comment (nwalters512) [inline apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache:87]: Can you include a screenshot of what this actually renders as?

Comment (eliotwrobson) [reply apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache:87]: I think it's the second screenshot here: https://github.com/PrairieLearn/PrairieLearn/pull/10433#issuecomment-2359230339

Comment (nwalters512) [reply apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache:87]: Lol, thanks, that's what I get for jumping straight to the changed files tab 😅

**Comment by eliotwrobson** [comment]
> My concern with the current implementation is that the "? missing input" thing looks _exactly_ like a button, but is not in fact one. Why do we need two things-that-look-like-buttons-and-have-question-mark-icons-in-them?
> 
> If this is what all other elements currently look like, I can approve this, but we should _definitely_ make that better in the future. This doesn't have to be a super-high priority, since like you said, this shouldn't be shown frequently.

This is what other elements look like. In agreement that the button shown is weird, which is why the invalid input message was changed to be less confusing.

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**